### PR TITLE
Notes final

### PR DESCRIPTION
**Pull request by tolyara**
Notes are ready, but photo uploading needs improvement

### CODE REVIEW

File: src/components/Main.js [hunk 1]
```diff
@@ -30,7 +32,7 @@ class Main extends Component {
             map:"",
             idPlan: '',
             mapComponent: '',
-            
+            idNote: 49
```
Comment (Djubabagdad) [inline src/components/Main.js]: ????

Comment (tolyara) [reply src/components/Main.js]: fixed

File: src/components/notesComponents/AuthorizedNotes.js [hunk 2]
```diff
@@ -5,78 +5,110 @@ import OnlyMyNotes from "./OnlyMyNotes";
...
+    setUserNotesAmount() {
+        axios.get(`http://localhost:8080/country/${this.props.countryName}/notes/private?user=${localStorage.getItem("login"
+        )}`).then(res => {
+            const notes = res.data;
+            console.log(notes.length);
+            localStorage.setItem('userNotesAmount', notes.length);
+        });
 
-        return (
+        // this.setState({
```
[the remark this answers is not in the document]
Comment (tolyara) [reply src/components/notesComponents/AuthorizedNotes.js]: done

**Comment by Djubabagdad** [comment]
Could you ,please, reopen this pull request.

**Pull request closed without merging**